pulp_file/tests/functional/api/test_crud_content_unit.py

```diff
@@ -180,6 +181,20 @@ def test_cannot_create_repo_version_with_two_relative_paths_the_same(
         monitor_task(response.task)
 
 
+@pytest.mark.parallel
+def test_bad_inputs_to_modify_endpoint(file_repo, file_repo_api_client, has_pulp_plugin):
+    has_pulp_plugin("core", min="3.23.0.dev")
```

Comment:
```suggestion
    needs_pulp_plugin("core", min="3.23.0.dev")
```
If you want the conditional skip.

Reply:
Oh, jeez, sorry.